Cesium.Preprocessor/CPreprocessor.cs

```diff
@@ -145,6 +145,16 @@ IToken<CPreprocessorTokenType> ConsumeNext(params CPreprocessorTokenType[] allow
 
                 return tokens.ToList();
             }
+            case "error":
+            {
+                bool hasRemaining;
+                var errorText = new StringBuilder();
+                while ((hasRemaining = enumerator.MoveNext()) && enumerator.Current is var t and not { Kind: NewLine })
+                {
+                    errorText.Append(t.Text);
+                }

```

Comment:
Since there shouldn't be any `NewLine` in the `enumerator` (`ReadDirectiveLine` drops it before this place is being called), this may be simplified.

```suggestion
                var errorText = new StringBuilder();
                while (enumerator.MoveNext())
                {
                    errorText.Append(enumerator.Current.Text);
                }

```

Reply:
Nice! Done. =)